docs/getting-started-tutorial.md

```diff
@@ -0,0 +1,101 @@
+
+For now, we will start out with a Postgres source and destination. Before we configure anything in the UI, we need databases and data. Run the following commands in a new terminal window to start backgrounded source and destination databases:
+
+```text
+docker run --rm --name airbyte-source -e POSTGRES_PASSWORD=password -p 2000:5432 -d postgres
+docker run --rm --name airbyte-destination -e POSTGRES_PASSWORD=password -p 3000:5432 -d postgres
+```
+
+Add a table to the source database:
+
+```text
+docker exec -it airbyte-source psql -U postgres -c "CREATE TABLE users(id SERIAL PRIMARY KEY, col1 VARCHAR(200));"
+```
+
+Run a loop which will add records to this table every few seconds:
+
+```text
+while true
+do
+    COL1=$(cat /dev/urandom | LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
+	docker exec -it airbyte-source psql -U postgres -c "INSERT INTO public.users(col1) VALUES('$COL1');"
+	sleep 3
+done
+```
+
+Return to the UI and configure a source Postgres database. Use the name `airbyte-source` for the name and `Postgres`as the type. Since Airbyte is running locally, we'll need to point to the Docker local host `host.docker.internal`. Fill in the configuration fields as follows:
+
+```text
+Host: host.docker.internal
+Port: 2000
+User: postgres
+Password: password
+DB Name: postgres
+```
+
+Click on `Set Up Source` and the wizard should move on to allow you to configure a destination. We currently support BigQuery, Postgres, and file output for debugging, but Redshift, Snowflake, and other destinations are coming soon. For now, configure the destination Postgres database:
+
+```text
+Host: host.docker.internal
+Port: 3000
+User: postgres
+Password: password
+DB Name: postgres
+```
+
+After adding the destination, you can choose what tables and columns you want to sync. For this demo we recommend leaving the defaults and selecting "Every 5 Minutes" as the frequency. Click `Set Up Connection` to finish setting up the sync.
+
+You should now see a list of sources with the source you just added. Click on it to find more information about your connection. This is the page where you can update any settings about this source and how it syncs. 
+
+There should be a `Completed` job under the history section. If you click on that run, it will show logs from that run. One of biggest problems we've seen in tools like Fivetran \(besides the price\) is the lack of visibility when debugging. In Airbyte, allowing full log access and the ability to debug and fix integration problems is one of our highest priorities. We'll be working hard to make these logs accessible and understandable. 
```

Comment:
```suggestion
There should be a `Completed` job under the history section. If you click on that run, it will show logs from that run. One of biggest problems we've seen in tools like Fivetran is the lack of visibility when debugging. In Airbyte, allowing full log access and the ability to debug and fix integration problems is one of our highest priorities. We'll be working hard to make these logs accessible and understandable. 
```